### ISSUE TITLE
Setting up CI

### ISSUE THREAD

**Issue by webmat** [elastic/ecs-mapper#15]
We should set up basic CI for this repo.

Things we could do as an MVP:

- [ ] Run `rake test`
- [ ] Run `rake example` & ensure no files have changed

**Comment by tonymeehan** [comment]
Agreed. I suspect this is also a dependency to publish a Gem from Elastic.
